test/addons/ocm-controller/kustomization.yaml

```diff
@@ -7,8 +7,7 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-# The default timeout (27 seconds) is too short when using a slow network.
-- https://github.com/stolostron/multicloud-operators-foundation.git/deploy/foundation/hub/overlays/ocm-controller?ref=main&timeout=300s
```

Comment:
Did you try to increase the timeout above? It worked when I tried it few months ago.

Reply:
Posted #1231

Reply:
Yes, I tried with 600 seconds. If you see the errors, it is not timing out but the connections are being terminated by the server. So the timeout increase doesn't help.

Reply:
It works: https://github.com/RamenDR/ramen/actions/runs/8194538533/job/22410806286?pr=1231

Reply:
Maybe it depends on the time of the run, and it could fail on the next run. Anyway we don't want to keep ocm-controller sources in ramen source. If kubectl apply does not work, lets try to clone the repo manually and install from the local repo, so we don't need to keep the sources and updated them.

Reply:
git clone is probably not what we want - but we can use curl or wget to get all the resources quickly without timeouts, and then deploy locally. Not sure why kustomize cannot do this reliably.